tests/scripts/kubeadm.sh

```diff
@@ -64,6 +99,29 @@ case "${1:-}" in
   clean)
     kubeadm_reset
     ;;
+  install)
+    sudo sh -c "${scriptdir}/kubeadm-install.sh ${KUBE_VERSION}" root
+    case "${2:-}" in
+        master)
+            install_master
+        ;;
+        node)
+            install_node $3 $4 $5
+        ;;
+        *)
+            echo "invalid install option" >&2
+            echo "usage: ">&2
+            echo "$0 $1 [master|node]" >&2
```

Comment:
The "node" command command takes 3 arguments, that should be reflected in the usage statement.  I'd suggest creating a usage() function that prints out the full usage, and then call that both here and down below in the wild-card case.  Also the "wait" command needs to be documented as taking an optional argument, and the code should make sure it is not passed more than one argument.

Reply:
Good idea,  implemented this suggestion.